sql/core/pom.xml

```diff
@@ -69,6 +69,11 @@
       <version>2.3.0</version>
     </dependency>
     <dependency>
+      <groupId>org.jodd</groupId>
+      <artifactId>jodd-core</artifactId>
+      <version>${jodd.version}</version>
+    </dependency>
```

Comment:
Okay, I talked to @pwendell and we think it would be better to use Joda time if possible since spark already depends on that library in other subprojects.  What do you think?